src/main/java/com/ase/lecturerservice/services/FeedbackService.java

```diff
@@ -35,4 +37,9 @@ public Exam getExam(String uuid) {
         .filter(exam -> exam.getUuid().equals(uuid))
         .findFirst().orElse(null);
   }
+
+  public void saveFeedback(Feedback feedback) {
+    feedbackRepository.save(feedback);
```

Comment:
```suggestion
	feedback.setUUID(null);
    feedbackRepository.save(feedback);
```
and then add an update methode below. Currently you could send a "POST /feedback" and override an existing feedback by providing the uuid of the feedback you want to override in the request body